.g8/datePage/generated-test/views/$className$ViewSpec.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import controllers.routes
+import forms.$className$FormProvider
+import models.{CheckMode, NormalMode}
+import play.api.i18n.Messages
+import play.api.mvc.Request
+import play.api.test.FakeRequest
+import views.html.$className$View
+
+import java.time.LocalDate
+
+
+class $className$ViewSpec extends ViewSpecHelper {
+
+  val view = application.injector.instanceOf[$className$View]
+  val formProvider = new $className$FormProvider
+  val form = formProvider.apply()
+  implicit val request: Request[_] = FakeRequest()
+
+  object Selectors {
+    val heading = "govuk-fieldset__heading"
+    val legend = "govuk-fieldset__legend  govuk-fieldset__legend--m"
+    val dateItems = "govuk-date-input__item"
+    val dateLables = "govuk-label govuk-date-input__label"
+    val errorSummaryList = "govuk-list govuk-error-summary__list"
+    val button = "govuk-button"
+    val form = "form"
+  }
+
+  "View" - {
+    val html = view(form, NormalMode)(request, messages(application))
+    val document = doc(html)
+    "should contain the expected title" in {
+      document.title() must include(Messages("$className;format="decap"$" + ".title"))
+    }
+
+    "should include a legend with the expected heading" in {
+      val legend = document.getElementsByClass(Selectors.legend)
+      legend.size() mustBe 1
+      legend.get(0).getElementsByClass(Selectors.heading).text() mustEqual Messages("$className;format="decap"$" + ".heading")
+    }
+
+    "when the form is not preoccupied and has no errors" - {
```

Comment:
I just want to verify "preoccupied" is the desired word choice.  I believe in other tests the wording is "pre-populated"